### DISCUSSION TITLE
Where to find documentation?

### DISCUSSION THREAD

**Discussion by Anixx** [ramensoftware/windhawk#157] [General]
I am trying to use
    WindhawkUtils::Wh_SetFunctionHookT(PrivateExtractIconsW, PrivateExtractIconsW_Hook, &PrivateExtractIconsW_Original);

It compiles but does not work. I even do not know what should it return. Can I find documentation on how to use it?

**Comment by m417z** [comment]
Documentation can be found here:
https://github.com/ramensoftware/windhawk/wiki

For more help, you can join Windhawk's Discord channel.

I assume you figured out the hook since you submitted the mod.
